package.json

```diff
@@ -98,7 +98,7 @@
     "eth-json-rpc-filters": "^4.1.1",
     "eth-json-rpc-infura": "^4.0.2",
     "eth-json-rpc-middleware": "^5.0.2",
-    "eth-keyring-controller": "^6.0.1",
+    "eth-keyring-controller": "github:ryanml/KeyringController#98bd0bd9655c304bdb7fe3fb93ca62a5e9e4f4d7",
```

Comment:
This will need to be updated when https://github.com/MetaMask/KeyringController/pull/71 is merged and the package is released

Reply:
v6.1.0 of the Keyring Controller has been published, so this can now be updated

Reply:
updated